docs/examples/data-access.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accessing Data\n",
+    "\n",
+    "There are many different conventions used to define unstructured grids (UGRID, SCRIP, etc.). These conventions often have different variable names used to describe the same variables.\n",
+    "\n",
+    "UXarray offers multiple ways of accessing data stored in `Grid` objects. This notebook will highlight different ways of accessing data with examples both in UGRID and non-UGRID formats.\n",
+    "\n",
+    "**Methods**\n",
+    "1. Indexing with Original Variable Name\n",
+    "2. UGRID Convenience Attributes\n",
+    "3. Indexing with Variable Dictionary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import uxarray as ux"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Base Data Path\n",
+    "base_path = \"../../test/meshfiles/\"\n",
+    "\n",
+    "# UGRID Grid File\n",
+    "ugrid_grid_path = base_path + \"outCSne30.ug\"\n",
+    "\n",
+    "# Non-UGRID Grid File\n",
+    "non_ugrid_grid_path = base_path + \"geoflow-small/grid.nc\""
```

Comment:
This is actually a UGRID file with different variable namings. We might need to modify the entire structure of the notebook.